utility/common.go

```diff
@@ -33,3 +33,14 @@ func InsertInMap(data map[string]interface{}, key string, values map[string]inte
 	}
 
 }
+
+func GetDeep(raw map[string]interface{}, keys ...string) map[string]interface{} {
```

Comment:
I think it is better GetDeep as a function than as a method, but right now they are duplicated (leftover i guess) so this one should be removed